CHANGELOG.md

```diff
@@ -2,11 +2,197 @@
+    - `WordPress.Arrays.ArrayDeclarationSpacing`: the fixer will now handle the new style annotations correctly.
+    - `WordPress.Arrays.CommaAfterArrayItem`: prevent a fixer loop when new style annotations are encountered.
+    - `WordPress.Files.FileName`: respect the new style annotations if these would selectively disable this sniff.
+    - `WordPress.WhiteSpace.ControlStructureSpacing`: handle the new style annotations correctly for the "blank line at the start/end of control structure" checks and prevent a fixer conflict when the new style annotations are encountered.
+    - `WordPress.WhiteSpace.PrecisionAlignment`: allow for checking of for precision alignment on lines containing new style annotations when `phpcs` is run with `--ignore-annotations`.
+- The `Sniff::is_test_class()` method now has improved recognition of namespaced test classes.
+    This positively affects the `WordPress.Files.FileName`, `WordPress.NamingConventions.PrefixAllGlobals` and `WordPress.WP.GlobalVariablesOverride` sniffs, which each allow for test classes to (partially) not comply with the rules these sniffs check for.
+    This fixes the following bugs:
+    - Namespaced classes where the classname was one of the whitelisted global classes would incorrectly be recognized as a test class, even though they were not the same class.
+        This also happened if a namespaced class `extend`ed one of the whitelisted global classes.
+    - A namespaced custom test class where the name was split between the namespace declaration and the extended class declaration was not correctly recognized as the whitelisted test class.
+    - A namespaced test class which extended another class using a FQCN prefixed with a `\\` would not be correctly recognized.
+    - The `custom_test_class_whitelist` property which is available for each of these sniffs expects to be passed a Fully Qualified Class Name. FQCNs prefixed with a global namespace indicator will now be correctly handled.
+- The determination of whether a `T_STRING` is a function call or not has been improved in the `AbstractFunctionRestrictions` class. This improvement benefits all sniffs which extend this abstract and any of its children (> 10 sniffs) and fixes the following false positives:
+    - Class declarations will no longer be confused with function calls.
+    - Use statement alias declarations will no longer be confused with function calls.
+- Various bugs in the `WordPress.Arrays.ArrayIndentation` sniff:
+    - The sniff will no longer throw false positives or try to fix multi-line text strings where the closing quote is on a line by itself.
+    - The sniff would go into a fixer loop when it encountered a multi-line trailing comment after an array item.
+- The `WordPress.CodeAnalysis.AssignmentInCondition` was throwing false positives for ternaries in nested, but unrelated, parenthesis.
```

Comment:
Parentheses should be plural since it does not have a determiner and is countable.